web/src/stream_pill.ts

```diff
@@ -16,7 +17,10 @@ export type StreamPillData = StreamSubscription & {type: "stream"};
 
 function display_pill(sub: StreamSubscription): string {
     const sub_count = peer_data.get_subscriber_count(sub.stream_id);
-    return "#" + sub.name + ": " + sub_count + " users";
+    return $t(
+        {defaultMessage: "#{stream_name}: {sub_count} users"},
+        {stream_name: sub.name, sub_count},
+    );
```

Comment:
I don't think this is right. The whole display value like "#stream: 15 users" can't be translated. Only "users" can be translated here. I think we don't translate numbers for now. To make this "#stream: 15 users" translation work, we'd have to create many combinations in the locale file with different stream names and numbers, which isn't practical. I think it should be something like:

```
function format_stream_name_and_subscriber_count(sub: StreamSubscription): string {
    const sub_count = peer_data.get_subscriber_count(sub.stream_id);
    return "#" + sub.name + ": " + sub_count + " " + $t({defaultMessage: "users"}, {sub_count},
    );
}
```

But this method isn't correct because we're adding a string with `${}` elements. To make translation work properly, I think we need to change the `InputPillItem` and modify the `display_value`. Instead of being just a string, `display_value` should be made up of a complex structure. This structure would allow us to separate translatable and untranslatable parts of the sentence.  However, I'm not sure if it's worth making such big and complicated changes. We should try to find a way to write some specific code for pills that can ignore certain parts when we translate them.

Reply:
Update: Separated "users" from the display value so that it could be translated properly.